json-rpc/types/src/views.rs

```diff
@@ -313,13 +313,37 @@ impl From<&Vec<u8>> for BytesView {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+pub struct VMStatusSourceContextView {
```

Comment:
What does this do?

Reply:
If there is a source-level error (e.g., `abort 5`) this is what will return that information out (`context: 0x1::Coin1, error: 4`). It is separated out from the major code since this does not need to exist always and is only for "source-derived" errors.

Reply:
Why is it just a string then? And why is it just one code and not two? Major and minor?

Reply:
Because this is being sent out over the wire to clients (e.g. someone `Curl`ing it). We could send hex bytes for these things if we wanted to....but I don't see the use for that really. For on-chain data structures, we do store this as a moduleid etc.

In terms of major/minor. The major is held in the `vm_status` view for the `TransactionView`. This struct is meant as a context for errors that have further information to provide.